include/librealsense2/utilities/CMakeLists.txt

```diff
@@ -2,7 +2,11 @@
 # Copyright(c) 2022 Intel Corporation. All Rights Reserved.
 
 include(${CMAKE_CURRENT_LIST_DIR}/easylogging/CMakeLists.txt)
+target_sources( ${PROJECT_NAME} PRIVATE ${ELPP_SOURCE_FILES} )
```

Comment:
Can't we place it inside 
`easylogging/CMakeLists.txt` ?

Same for `concurrency/CMakeLists.txt`.

Reply:
Not from what I remember, there was a problem.
I can try for the next PR.